src/main/java/com/conveyal/datatools/manager/jobs/MonitorServerStatusJob.java

```diff
@@ -161,14 +149,9 @@ public void jobLogic() {
                 .withTargetGroupArn(otpServer.ec2Info.targetGroupArn)
                 .withTargets(new TargetDescription().withId(instance.getInstanceId()));
             boolean targetAddedSuccessfully = false;
-
             // Wait for two minutes for targets to register.
             TimeTracker registerTargetTracker = new TimeTracker(2, TimeUnit.MINUTES);
-            AmazonElasticLoadBalancingClientBuilder elbBuilder = AmazonElasticLoadBalancingClient.builder()
-                .withCredentials(credentials);
-            AmazonElasticLoadBalancing elbClient = deployJob.getCustomRegion() == null ?
-                elbBuilder.build() :
-                elbBuilder.withRegion(deployJob.getCustomRegion()).build();
+            AmazonElasticLoadBalancing elbClient = deployJob.getELBClientForDeployJob();
```

Comment:
I would prefer to maybe have the elbClient and ec2Client as instance variables that are set to deployjob's instances in the monitor job's constructor.

Reply:
That would result in a problem of having stale clients for clients that depend on obtaining a session.

Reply:
In that case, having a comment about that would be helpful. Both here and on the deployjob's methods for ec2 and elb clients.

Reply:
There are 13 places where an EC2 client is potentially created in the DeployJob class alone, so I'm just going to add a note here for the ELB client.